include/RAJA/policy/openmp/policy.hpp

```diff
@@ -42,10 +60,23 @@ struct For {
 struct NoWait {
 };
 
-template <unsigned int ChunkSize>
-struct Static : std::integral_constant<unsigned int, ChunkSize> {
+static constexpr int default_chunk_size = -1;
```

Comment:
This is a *public* interface option for users. The OpenMP runtime says that a "default" will be used when a passed value into `omp_set_schedule` is negative